pkg/imgutil/imgutil.go

```diff
@@ -413,3 +415,32 @@ func ParseRepoTag(imgName string) (string, string) {
 
 	return repository, tag
 }
+
+// ValidateOutputPath validates the output paths.
+// Check whether `path` points to a regular file
+func ValidateOutputPath(path string) error {
+	if fileInfo, err := os.Stat(path); !os.IsNotExist(err) {
+		if err != nil {
+			return err
+		}
+
+		if fileInfo.Mode().IsRegular() {
+			return nil
+		}
+		if err := ValidateOutputPathFileMode(fileInfo.Mode()); err != nil {
+			return fmt.Errorf("invalid output path: %q must be a regular file: %w", path, err)
```

Comment:
Why?

Reply:
Historically, tar balls were designed to be directly writable to tape devices in 19XX.

And it is still 100% legit to write tar balls to device files in 2021. (Rarely useful, though)